integration-test/background/atb.js

```diff
@@ -141,6 +151,10 @@ describe('install workflow', () => {
         })
 
         it('should retreive stored ATB value on reload', async () => {
+            if (manifestVersion === 3) {
```

Comment:
There's a FIXME for MV3 comment below, I wonder if you addressed that we could leave this test case enabled?

Reply:
I tried, the test just errors. I'm not sure that `runtime.reload()` works.